optuna/integration/allennlp.py

```diff
@@ -26,11 +26,13 @@ def dump_best_config(input_config_file: str, output_config_file: str, study: opt
 
     Args:
         input_config_file:
-            Input configuration file used with :class:`~optuna.integration.AllenNLPExecutor`.
+            Input configuration file (Jsonnet) specified in
+            :class:`~optuna.integration.AllenNLPExecutor`.
         output_config_file:
-            Output configuration file.
+            Output configuration (Json) file.
```

Comment:
A nit but I think this is more common.
```suggestion
            Output configuration (JSON) file.
```

Reply:
I'd personally remove the parenthesis here, mostly a matter of style.